### DISCUSSION TITLE
[Suggestion] Separate config for Honey Blocks

### DISCUSSION THREAD

**Discussion by RaidhoSketch** [Roadhog360/Et-Futurum-Requiem#334] [Improvements]
Currently, only one config disables all bee-related content - including honey blocks. We ask for a separate config for just enabling honey blocks without the bee branch - this is mainly applicable when someone already has honey and bees (like, say, with Forestry), or just wants to play with pistons in creative.

**Comment by Roadhog360** [comment]
Understandable. I will admit having them all be one config option was based on a short-sighted assumption. I will add the honeycomb block and honey blocks, as well as honey bottles to separate config toggles in 2.5.1. They will still be craftable if you have modded honey that uses the correct tags.

**Comment by Roadhog360** [comment]
This was added in 2.5.1
